ui/src/common/utils.js

```diff
@@ -251,3 +251,7 @@ export function getAuthorName(author) {
   }
   return author.get('full_name');
 }
+
+export function addCommasToNumber(number) {
+  return number && number.toLocaleString();
```

Comment:
this doesn't do `add commas to number` necessarily, the behavior of this function depends on the user's locale so in `China` or `UAE` this will return differently.

if what we want is to format the number according to the user locale, then that's fine we can just change the function name, but I don't think that's case. Because we don't intl. anywhere in our app.

So it would be better to pass the locale explicitly, most likely `en-US`